src/Compilers/Lua/Portable/Parser/Lexer.cs

```diff
@@ -28,6 +29,7 @@ private struct TokenInfo
             internal double DoubleValue;
             internal uint UIntValue;
             internal long LongValue;
+            internal ulong ULongValue;
```

Comment:
Create below needs to be modified to create the integer token with this new value.